nuxt.config.ts

```diff
@@ -112,7 +112,20 @@ const config: NuxtConfig = {
     'nuxt-lazy-load'
   ],
 
-  components: true,
+  components: [
+    '~/components/advocates',
+    '~/components/blockbuster',
+    '~/components/events',
+    '~/components/global',
+    '~/components/landing',
+    '~/components/layouts',
+    '~/components/learn',
+    '~/components/logic',
+    '~/components/metal',
+    '~/components/overview',
+    '~/components/textbook-beta',
+    '~/components/ui'
+  ],
```

Comment:
I would like someone to confirm that this covers all cases.

Reply:
By using this way of register components, nuxt includes prefixes and the nested components are not shown. I've modified the configuration to set the `pathPrefix: false`